web/models/abilities.ex

```diff
@@ -0,0 +1,17 @@
+defimpl Canada.Can, for: Ex338.User do
+  alias Ex338.{User}
+
+  def can?(%User{admin: true }, _, _), do: true
+
+  def can?(%User{id: user_id }, action, draft_pick)
```

Comment:
There is no whitespace around parentheses/brackets most of the time, but here there is.